examples/sample_measurement/measurement.py

```diff
@@ -42,6 +43,11 @@ class Color(Enum):
 @measurement_service.configuration("Bool In", nims.DataType.Boolean, False)
 @measurement_service.configuration("String In", nims.DataType.String, "sample string")
 @measurement_service.configuration("Enum In", nims.DataType.Enum, Color.BLUE, enum_type=Color)
+@measurement_service.configuration(
```

Comment:
The client generator doesn't support message configurations: https://github.com/ni/measurement-plugin-python/blob/main/packages/generator/ni_measurement_plugin_sdk_generator/client/_support.py#L126

Reply:
You should also update https://github.com/ni/measurement-plugin-python/blob/main/packages/generator/tests/utilities/measurements/non_streaming_data_measurement/__init__.py in order to test this.